docs/CONTRIBUTING.md

```diff
@@ -128,9 +128,9 @@ component once your proposal is approved.
 ### Implement a proposal
 
 All components should start with a proposal. One of our
-[Guiding Principles](#guiding-principles) is to start all components with
-documentation. Once your proposal has been approved, proceed to contribute code
-to Atlantis to make it a reality!
+[Guiding Principles](https://atlantis.getjobber.com/?path=/docs/contributing--docs#guiding-principles)
```

Comment:
This link wasn't working from my testing. I updated this to point to the location on the atlantis domain because it was only opening in the iframe or not scrolling when I changed it to match the others

Reply:
hmmm I'm still seeing some weirdness with the link. I'm ok with just removing this link since it's referring to something directly below, if we can't get it working as intended.
https://github.com/user-attachments/assets/55944442-459a-4b97-a8c6-e4b0c19d2675

Reply:
Sure thing, I noticed that our Table of contents doesn't use default browser behavior (it does a javascript scroll). I will update it right now because I am pretty sure it is a storybook router issue

Reply:
Removed in 464ff26e

Reply:
The `InputDate` story has a 'see related components' link that works 🤔

Reply:
It is an link to another page where as Guiding Principles is a link to the same page but a different section

Reply:
ohh I see let me see. My assumption is mdx vs MD

Reply:
Same thing for the link to `InputText` on `InputPassword` story - but this should be able to be like all the other links?

https://github.com/user-attachments/assets/872a856c-abb2-42b3-9971-d506d7fc8411

Reply:
So looking at it the internal links on component pages have a `sb-docs` class but the contributing.md link doesn't 🤔 
The InputPassword is a fun one (it links to the live atlantis docs page directly instead of locally 😭  )